connector/src/main/java/org/geysermc/connector/network/translators/world/WorldManager.java

```diff
@@ -71,4 +74,18 @@ public int getBlockAt(GeyserSession session, Vector3i vector) {
      * @return the block state at the specified location
      */
     public abstract int getBlockAt(GeyserSession session, int x, int y, int z);
+
+    public abstract void setGameRule(GeyserSession session, String name, Object value);
```

Comment:
When you're done coding, some Javadoc comments might be nice here? They seem relatively descriptive, so maybe just `hasPermission`.